src/project-tests/portfolio-tests.js

```diff
@@ -81,7 +81,11 @@ export default function createPortfolioTests() {
       it(`${reqNum}. The navbar should contain at least one link that I can
       click on to navigate to different sections of the page.`,
       function(done) {
+        // Filter anchors by location hash (don't click external links)
         const links = Array.from(document.querySelectorAll('#navbar a'));
+        links.filter(function(nav) {
```

Comment:
I'm not sure this is working the way you intend. I think `filter` does not change the original array. I think you have to do something like this:

```
links = links.filter(function(nav) {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter